packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -0,0 +1,23 @@
+import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
+import { Script } from '@defichain/jellyfish-transaction/tx'
+import { CScript } from '@defichain/jellyfish-transaction/tx_composer'
+
+// Disabling no-return-assign makes the code cleaner with the setter and getter */
+/* eslint-disable no-return-assign */
+
+export interface CreateVault {
+  script: Script
+  loanSchemeId: string
```

Comment:
add this item too https://github.com/DeFiCh/ain/blob/d62944b54db3a15d68350330fd26f95114b573c9/src/masternodes/loan.h#L185